frontend/tsconfig.json

```diff
@@ -8,18 +8,22 @@
     ],
     "allowJs": true,
     "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Out of curiosity, why is strict mode disabled?

Reply:
I don't know 🤔 This was default in NextJS so I'll change it back to true 💪